chain/chain/src/store.rs

```diff
@@ -2929,8 +3006,10 @@ mod tests {
             DBCol::ColNextBlockWithNewChunk,
             DBCol::ColChunkPerHeightShard,
             DBCol::ColBlockRefCount,
+            DBCol::ColOutcomesByBlockHash,
         ];
         for i in DBCol::iter() {
+            println!("{:?}", i);
```

Comment:
Remove debug output

Reply:
why?

Reply:
Why do we need this println?

Reply:
It prints which column is processing now. If assert fails, it would be clear what's wrong even without local rerun.

Reply:
Then I would prefer having some accompanying message along side to the printed value then

Reply:
Makes sense, ++.
BTW, I would like to point that printing some minor debug in tests is a good thing. I don't know why we used to avoid it.

Reply:
I believe, we should better invest into having enough debug tracing calls in the code, so we don’t need to have extra debug information in the tests. This way we will be able to debug issues on ruining instances, and not only in tests.

Reply:
It's romantic. In fact, when I look into failing catching_up or gc test, only manual tracing creates most of understanding. 99% of standard debug is useless in those cases - not because it's useless overall, but because the test checks some specific things that need to be tested.